contracts/scripts/deploy.js

```diff
@@ -8,26 +8,26 @@ require("dotenv").config({ path: path.resolve(__dirname, "../.env") });
 const SANDBOX_URL = "http://127.0.0.1:20000";
 const TESTNET_URL = "https://rpc.tzkt.io/hangzhou2net";
 
-const FAUCET = JSON.parse(process.env.FAUCET);
-const ALICE_SECRET = process.env.ALICE_SECRET;
 
 const deploy = async () => {
   const mode = process.argv[2] || "sandbox";
   console.log(">>> Deploy with mode:", mode);
-
+  
+  var Tezos;
   try {
-    const Tezos = new TezosToolkit(
-      mode === "sandbox" ? SANDBOX_URL : TESTNET_URL
-    );
-
-    // Signing on testnet mode needs different parameters as on sandbox mode
-    const { email, password, mnemonic, secret } = FAUCET;
-    mode === "sandbox"
-      ? await importKey(Tezos, ALICE_SECRET)
-      : await importKey(Tezos, email, password, mnemonic.join(" "), secret);
-
+    if (mode === "sandbox") {
+      const ALICE_SECRET = process.env.ALICE_SECRET;
+      Tezos = new TezosToolkit(SANDBOX_URL);
+      await importKey(Tezos, ALICE_SECRET)
+    } else {
+      const FAUCET = JSON.parse(process.env.FAUCET);
+      const { email, password, mnemonic, secret } = FAUCET;
+      Tezos = new TezosToolkit(TESTNET_URL);
+      await importKey(Tezos, email, password, mnemonic.join(" "), secret);
+    }
+    
     const code = fs
-      .readFileSync(path.resolve(__dirname, "../contracts/out/taia_x_main.tz"))
+      .readFileSync(path.resolve(__dirname, "../contracts/src/taia_x_main.mligo"))
```

Comment:
Is this correct to be the `mligo` files?

Reply:
@amrap030 yes, exactly after talking with @repype, we found this was the problem with the deployment issue. The compiled contract should be deployed not the ligo code